api/libproject/devstart.rb

```diff
@@ -2580,3 +2580,71 @@ def start_api_and_incremental_build(cmd_name, args)
   :description => "Used internally by other commands.",
   :fn => ->(*args) { start_api_and_incremental_build("start-api-and-incremental-build", args) }
 })
+
+def randomize_vcf(cmd_name, *args)
+  op = WbOptionsParser.new(cmd_name, args)
+  op.add_option(
+      "--vcf [vcf]",
+      -> (opts, v) {opts.vcf = v},
+      "Input vcf to randomize. This vcf should be gzipped and its corresponding index file " +
+          "should be in the same folder. The index file can be generated from a gzipped vcf " +
+          "by running tabix -p vcf [vcf path]"
+  )
+  op.add_option(
+      "--number-of-copies [n]",
+      -> (opts, n) {opts.n = n},
+      "How many random vcfs to generate."
+  )
+  op.add_option(
+      "--output-dir [out]",
+      -> (opts, o) {opts.out = o},
+      "Output dir in which to put randomized vcfs"
+  )
+  op.parse.validate
+
+  for i in 1..op.opts.n.chomp.to_i
+    basename = File.basename(op.opts.vcf, ".vcf.gz")
+    out = "#{op.opts.out}#{basename}.#{i}.vcf"
+    app_args = "-PappArgs=['-V#{op.opts.vcf}','-O#{out}','-S#{i}']"
+    Common.new.run_inline %W{./gradlew -p genomics randomizeVcf} + [app_args]
+  end
+end
+
+Common.register_command({
+  :invocation => "randomize-vcf",
+  :description => "Given an example vcf and a number of copies to make, generates that many " +
+    "random copies in a given output directory",
+  :fn => ->(*args) { randomize_vcf("randomize-vcf", *args) }
+})
+
+def combine_vcfs(cmd_name, *args)
+  op = WbOptionsParser.new(cmd_name, args)
+  op.add_option(
+      "--dir [dir]",
+      -> (opts, d) {opts.dir = d},
+      "Dir with vcfs to combine. vcfs should have corresponding index files in the same " +
+          "folder."
+  )
+  op.add_option(
+      "--output [output]",
+      -> (opts, o) {opts.output = o},
+      "Path to output the combined vcf"
+  )
+  op.parse.validate
+
+  app_args = "-PappArgs=["
```

Comment:
opt: We have several other commands that need to prep args like this, I think the pattern could be reused here - or it may not be relevant if the flag construction for GATK is specialized:
https://github.com/all-of-us/workbench/blob/0ccccd01288d7faeaf3787e677ecde7019274482/api/libproject/devstart.rb#L1341

probably there's enough instances to pull it into a helper eventually.

Reply:
the flag construction for GATK is different than we do it and the flag construction for picard is differently different...